scripts/cave8.lua

```diff
@@ -97,11 +97,13 @@ function entity_handler(en)
         set_ent_facing(HERO2, FACE_DOWN)
         x, y = marker("nostik_e")
         move_entity(1, x, y, 0)
-        wait_for_entity(1, 2)
+        wait_for_entity(HERO1, 1)
```

Comment:
This commit looks like it should be the `cmake` stuff. Could you move these changes to a separate commit? (It doesn't have to be moved to a new pull request.)